css/fieldmanager-richtext-content.css

```diff
@@ -1,144 +0,0 @@
-del {color:red; text-decoration:line-through}
-cite {border-bottom:1px dashed blue}
-acronym {border-bottom:1px dotted #CCC; cursor:help}
-abbr {border-bottom:1px dashed #CCC; cursor:help}
-
-img:-moz-broken {-moz-force-broken-image-icon:1; width:24px; height:24px}
-font[face=mceinline] {font-family:inherit !important}
-*[contentEditable]:focus {outline:0}
-
-.mceItemMedia {border:1px dotted #cc0000; background-position:center; background-repeat:no-repeat; background-color:#ffffcc}
-.mceItemShockWave {background-image:url(../../img/shockwave.gif)}
-.mceItemFlash {background-image:url(../../img/flash.gif)}
-.mceItemQuickTime {background-image:url(../../img/quicktime.gif)}
-.mceItemWindowsMedia {background-image:url(../../img/windowsmedia.gif)}
-.mceItemRealMedia {background-image:url(../../img/realmedia.gif)}
-.mceItemVideo {background-image:url(../../img/video.gif)}
-.mceItemAudio {background-image:url(../../img/video.gif)}
-.mceItemIframe {background-image:url(../../img/iframe.gif)}
-.mcePageBreak {display:block;border:0;width:100%;height:12px;border-top:1px dotted #ccc;margin-top:15px;background:#fff url(../../img/pagebreak.gif) no-repeat center top;}
-
-/* WordPress styles */
-body {
-	font-family: Georgia, "Times New Roman", "Bitstream Charter", Times, serif;
-	font-size: 13px;
-	line-height: 19px;
-	color: #333;
-	margin: 10px;
-	min-height: 100%;
-}
-
-br[data-mce-bogus] {
-	line-height: 1em;
-	margin-top: -1em;
-}
-
-br[data-mce-bogus]:only-child {
-	line-height: inherit;
-	margin-top: inherit;
-}
-
-.aligncenter,
-dl.aligncenter {
-	display: block;
-	margin-left: auto;
-	margin-right: auto;
-}
-
-.alignleft {
-	float: left;
-}
-
-.alignright {
-	float: right;
-}
-
-.wp-caption {
-	border: 1px solid #ddd;
-	text-align: center;
-	background-color: #f3f3f3;
-	padding-top: 4px;
-	margin: 10px 0;
-	-webkit-border-radius: 3px;
-	border-radius: 3px;
-}
-
-.mceIEcenter {
-	text-align: center;
-}
-
-.wp-caption img {
-	margin: 0;
-	padding: 0;
-	border: 0 none;
-	-webkit-user-drag: none;
-}
-
-.wp-caption-dd {
-	font-size: 11px;
-	line-height: 17px;
-	padding: 0 4px 5px;
-	margin: 0;
-}
-
-pre {
-	font: 12px/18px Consolas, Monaco, monospace;
-}
-
-td {
-	color: #000;
-	font-size: 11px;
-	margin: 8px;
-}
-
-/* Styles for the WordPress plugins */
-img.mceWPnextpage,
-img.mceWPmore {
-	border: 0;
-	border-top: 1px dotted #cccccc;
-	display: block;
-	width: 95%;
-	height: 12px;
-	margin: 15px auto 0;
-}
-
-img.mceWPmore {
-	background: transparent url("img/more_bug.gif") no-repeat right top;
-}
-
-img.mceWPnextpage {
-    background: transparent url("img/page_bug.gif") no-repeat right top;
-}
-
-img.wpGallery {
-	border: 1px dashed #888;
-	background: #f2f8ff url("img/gallery.png") no-repeat scroll center center;
-	width: 99%;
-	height: 250px;
-}
-
-img.wp-oembed {
-	border: 1px dashed #888;
-	background: #f7f5f2 url("img/embedded.png") no-repeat scroll center center;
-	width: 300px;
-	height: 250px;
-}
-
```

Comment:
Is this change intentional? I would expect these build workflow changes to leave existing styles in place (aside from the vendor prefix removals).

Reply:
Yes, as far as I could tell this was a stale file and was not being used.